docs/genindex.html

```diff
@@ -281,172 +223,34 @@ <h2 id="D">D</h2>
   </dt>
 
       
-  <dt><a href="dynamic_rest.html#dynamic_rest.fields.DynamicComputedField">DynamicComputedField (class in dynamic_rest.fields)</a>
```

Comment:
For some reason `make docs` was failing with `django.core.exceptions.AppRegistryNotReady`. Fixed this problem and regenerated docs.